sway/commands.c

```diff
@@ -1124,9 +1124,26 @@ static struct cmd_results *cmd_bar(int argc, char **argv) {
 	bar->tray_padding = config->bar.tray_padding;
 	list_add(config->bars, bar);
 
+	// set bar id
+	int i, n;
+	int len = 5; // "bar-" + \0
```

Comment:
Nevermind, was me being stupid. I think there's a function in here somewhere that tells you how big a number is, we should reuse it.